src/components/Header/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC, useState } from "react";
+import { CATEGORIES } from "@constants/categories";
+import Dropdown from "@components/Dropdown";
+import "./Navigation.scss";
+
+export const Navigation: FC = () => {
+  const [activeCategory, setActiveCategory] = useState<string | null>(null);
+  const [activeBrand, setActiveBrand] = useState<string | null>(null);
+
+  console.log(activeBrand); // temporarily 
+
+  return (
+    <ul className="navigation">
+      {CATEGORIES.map((category) => (
+        <li
+          key={`nav-${category.name}`}
+          className="navigation__item"
+          onMouseEnter={() => setActiveCategory(category.name)}
+          onMouseLeave={() => setActiveCategory(null)}
+        >
+          <span>{category.name}</span>
+          {activeCategory === category.name && (
+            <Dropdown items={category.brands} onSelect={setActiveBrand}/>
```

Comment:
Where is green arrow for category ?
And think it's too big on mobile

Also pay attention on styles
![image](https://github.com/NikitaIASA/freshnesekom-reenbit/assets/53089462/42fe9f2f-4205-4f24-8be1-6a07e5b5df1a)
![image](https://github.com/NikitaIASA/freshnesekom-reenbit/assets/53089462/5188c445-2df0-47cf-ac51-57bb755fda94)